android_emulator_cli.sh

```diff
@@ -31,7 +34,42 @@ echo; echo;
 echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
 echo "Welcome to the Android Emulator CLI only script!"
 echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
-sleep .5
+
+sleep 1
+
+echo; echo;
+echo "**** REQUIRED ****   Have you 'Requested administer access' yet before installation? (1/2)   **** REQUIRED ****"
+echo; echo;
+select yn in "Yes" "No"; do
+	case $yn in
+		Yes ) break;;
+		No ) echo "Exiting"; exit;;
+		* ) echo "Error: Did not recognize input. Please enter 1 or 2.";;
+	esac
+done
```

Comment:
Still not sure if this is required (or contributes to installation errors) but kind of feels like it should be there, especially if IT changes more installation privileges.